web/src/components/RunDetailTrace/AnalyzerPanel.tsx

```diff
@@ -12,7 +12,7 @@ const panel = {
   name: 'ANALYZER',
   maxSize: 650,
   minSize: 25,
-  isDefaultOpen: true,
+  isDefaultOpen: false,
```

Comment:
Do we want this?

Reply:
I was testing it, not sure if we want it or not tbh, will leave it open as it is today and discuss it with the team on Thurs